manifests/master.pp

```diff
@@ -389,11 +389,26 @@
       notify  => Service[$service]
     }
 
+    # `[master] ca` is deprecated, as of Puppet 5.5.6 (SIMP-5456), and removed
+    # in Puppet 6 (PUP-9158).  All
+    if versioncmp($facts['puppetversion'], '6.0') >= 0 {
+      $_ensure_master_ca = 'absent'
+    }
+    elsif versioncmp($facts['puppetversion'], '5.5.6') {
```

Comment:
According to https://puppet.com/docs/puppet/5.5/lang_conditional.html#if-statements the condition of an if statement should resolve to a boolean.  Would be more clear if this elsif resolved to a boolean, instead of a number